Documentation/gettingstarted/k8s-install-default.rst

```diff
@@ -0,0 +1,31 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    http://docs.cilium.io
+
+.. _k8s_quick_install:
+
+******************
+Quick Installation
+******************
+
+This guides takes you through the quick installation procedure.  The default
+settings will store all required state using Kubernetes custom resource
+definitions (CRDs). This is the simplest installation method as it only depends
+on Kubernetes and does not require additional external dependencies. It is a
+good option for environments up to about 250 nodes. For bigger environments or
```

Comment:
I was under the impression that it was scalable up until 100 nodes? Where is this number coming from?

Reply:
250 is a guess. We don't really know as it depends on the churn x number-of-nodes. I think we'll learn quickly and adjust the documentation as we get feedback.